web/src/portico/landing-page.ts

```diff
@@ -199,19 +256,21 @@ $(() => {
     if (window.location.pathname === "/team/") {
         const contributors = page_params.contributors;
         delete page_params.contributors;
+        assert(contributors !== undefined, "Expect contributors in page_params");
         render_tabs(contributors);
     }
 
     // Source: https://stackoverflow.com/questions/819416/adjust-width-and-height-of-iframe-to-fit-with-content-in-it
     // Resize tweet to avoid overlapping with image. Since tweet uses an iframe which doesn't adjust with
     // screen resize, we need to manually adjust its width.
 
-    function resizeIFrameToFitContent(iFrame) {
+    function resizeIFrameToFitContent(iFrame: HTMLIFrameElement): void {
         $(iFrame).width("38vw");
     }
 
     window.addEventListener("resize", () => {
-        const iframes = document.querySelectorAll(".twitter-tweet iframe");
+        const iframes: NodeListOf<HTMLIFrameElement> =
+            document.querySelectorAll(".twitter-tweet iframe");
```

Comment:
We can use `document.querySelectorAll<HTMLIFrameElement>(".twitter-tweet iframe");` here.